docs/content-services/components/search-date-range-advanced-tabbed.component.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: Search date range advanced component
+Added: v6.1.0
+Status: Active
+Last reviewed: 2023-06-13
+---
+
+# [Search date range advanced tabbed component](../../../lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.ts "Defined in search-date-range.component.ts")
```

Comment:
Should be probably: 
"Defined in search-date-range-advanced-tabbed.component.ts"

Reply:
Corrected the typo. Thanks!!